swagger/api/controllers/cities.js

```diff
@@ -0,0 +1,51 @@
+var City = require('../models/mongo/cities'); 
+
+const getRandomCity = (req, res) => {
+  City.find({}, (err, cities) => {
+    if (err) return res.send(err);
+    const cityIndex = Math.round(Math.random() * (cities.length - 1));
+    return res.send(cities[cityIndex]);
+  });
+};
+
+const getAllCities = (req, res) => {
+  City.find({}, (err, cities) => {
```

Comment:
Current version of Mongoose supports promises, i.e. CRUD methods return promises. Therefore the better way is to fully use ES6+ javascript and avoid callbacks.
`City.find({}).then(.........).catch(......)`